api/src/test/java/org/cardanofoundation/rosetta/api/construction/service/CardanoConstructionServiceImplTest.java

```diff
@@ -94,6 +98,26 @@ void outputMoreThanInputTest() {
     assertEquals("The transaction you are trying to build has more outputs than inputs", exception.getError().getMessage());
   }
 
+  @Test
+  void calculateTxSize_whenUnsignedTransactionAddressesInvalid_thenThrowException() {
+    List<Operation> operations = Collections.singletonList(givenOperation());
+    try (MockedStatic<CardanoAddressUtils> mockedAddressUtil = Mockito.mockStatic(
+        CardanoAddressUtils.class)) {
+      mockedAddressUtil.when(() -> CardanoAddressUtils.getEraAddressType(anyString()))
+          .thenReturn(null);
+      mockedAddressUtil.when(() -> CardanoAddressUtils.isEd25519KeyHash(anyString()))
```

Comment:
This mock is not reachable

Reply:
Why? This mock is needed for the #extractSignaturesFromAddress method
![image](https://github.com/cardano-foundation/cardano-rosetta-java/assets/167304900/e72be326-1af5-472b-932f-a77be180b875)

Reply:
Added verify() check